Sustainsys.Saml2/Metadata/KeyInfo.cs

```diff
@@ -113,7 +113,7 @@ public SecurityKeyIdentifier MakeSecurityKeyIdentifier()
 				{
 					ski.Add(new DsaKeyIdentifierClause(dsaKeyValue.Parameters));
 				}
-#if !NET461
+#if !NET462
```

Comment:
these compiler directives for NET461 were changed to NET462 because target framework was changed to NET462